libr/util/log.c

```diff
@@ -2,32 +2,34 @@
 
 #include <r_util.h>
 
-// XXX: This api is kinda ugly.. we need to redefine it
+R_API void r_vlog(RLogLevel level, const char *fmt, va_list ap) {
+	const char *hdr = "";
 
-static const char *logfile = "radare.log";
-
-R_API void r_log_file(const char *str) {
-	FILE *fd = r_sandbox_fopen (logfile, "a+");
-	if (fd) {
-		fputs (str, fd);
-		fclose (fd);
-	} else {
-		eprintf ("ERR: Cannot open %s\n", logfile);
+	switch (level) {
```

Comment:
index inside an array of strings, dont abuse a switch case for this

Reply:
ok but I see no real advantage.